doc-content/enterprise-only/admin-and-config/exporting-importing-dashbuilder-data-proc.adoc

```diff
@@ -13,15 +13,20 @@ IMPORTANT: This feature is only accessible by administrator users.
 
 == Exporting Dashbuilder data
 
-You can export all Dashbuilder data from {CENTRAL} as a ZIP file.
+You can export the dashbuilder related data such as datasets, perspectives, and navigation from {CENTRAL} as a ZIP file.
 
 .Procedure
 . In {CENTRAL}, select the *Admin* icon in the top-right corner of the screen and select *Dashbuilder Data Transfer*.
 
-. On the *Dashbuilder Data Transfer* page, click *Export*.
-+
-An `export.zip` file containing all Dashbuilder data is downloaded. The `export.zip` file structure is separated by data type. For example:
+. To export the dashbuilder related data, do any of the following tasks:
 +
+--
+If you want to export all dashbuilder data as a ZIP file, do the following tasks:
```

Comment:
I think the following text will work for restructuring the nested layout (please undo any changes you disagree with!):

```
== Exporting Dashbuilder data

You can export Dashbuilder related data such as datasets, perspectives, and navigation from {CENTRAL} as a ZIP file.

.Procedure
. In {CENTRAL}, select the *Admin* icon in the top-right corner of the screen and select *Dashbuilder Data Transfer*.

. To export the Dashbuilder related data, do any of the following tasks:
+
--
* If you want to export all Dashbuilder data as a ZIP file, on the *Dashbuilder Data Transfer* page, click *Export all*.
+
An `export.zip` file containing all Dashbuilder data is downloaded. The `export.zip` file structure is separated by data type. For example:
+
[source]
----
dashbuilder/datasets/definitions/dataset-example1.csv
dashbuilder/datasets/definitions/dataset-example1.dset
dashbuilder/datasets/definitions/dataset-example2.csv
dashbuilder/datasets/definitions/dataset-example2.dset
dashbuilder/datasets/readme.md
dashbuilder/perspectives/page1/perspective_layout
dashbuilder/perspectives/page1/perspective_layout.plugin
dashbuilder/perspectives/page2/perspective_layout
dashbuilder/perspectives/page2/perspective_layout.plugin
dashbuilder/perspectives/readme.md
dashbuilder/navigation/navigation/navtree.json
dashbuilder/navigation/readme.md
VERSION
----

* If you want to export customized user created Dashbuilder data as a ZIP file, do the following tasks:

. On the *Dashbuilder Data Transfer* page, click *Custom export*.
. In the *Export Wizard* panel, select the datasets that you want to include in the ZIP file and click *Next*.
. Select the pages that you want to include in the ZIP file and click *Next*.
+
The *Export Wizard* panel validates the selected datasets and pages. The summary of the datasets and pages is available on the panel.
+
NOTE: Navigation is always included in the exported ZIP file.

. Click *Download*.
+
An `export.zip` file containing customized Dashbuilder data is downloaded.
+
NOTE: You must select the pages and the datasets associated with those pages in the *Export Wizard* panel. You must select at least one page. If you fail to select both datasets and pages, an export error is generated and you can not download the `export.zip` file.

. Click *Finish*.
. Optionally, click *Cancel* to return to the *Dashbuilder Data Transfer* page.

--
```

Reply:
Thank you for this extensive suggestion. 
Actually I consulted Stetson's feedback as well and showed him the review. He said if you remove the bullet point then it will fit according to the style guide. 
But he also said it is his personal choice so now I am confused whether to use bullet point or not :)